src/TLuaInterpreter.cpp

```diff
@@ -245,20 +254,23 @@ void TLuaInterpreter::handleHttpOK(QNetworkReply* reply)
         QString localFileName = downloadMap.value(reply);
         downloadMap.remove(reply);
 
-        // If the user forgot to give us a file path, we're not going to
-        // consider this an error, we're just going to skip downloading the
-        // response. Another way this could happen is the user made a POST
+        // If the user did not give us a file path, we're not going to
+        // consider this an error, we're just going attach the reply
+        // directly. Another way this could happen is the user made a POST
         // request, and it redirected to a GET. In the case of POST requests,
-        // we don't ask the user for a file path, and so here too we will just
-        // skip downloading the response.
+        // we don't ask the user for a file path.
         if (localFileName.isEmpty())
```

Comment:
Not your fault but our coding style should have the opening `{` on this line and not the next one...